<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget#Form
{
	background-color: rgba(249, 249, 249, 255);
}


QWidget
{
	background-color: rgba(239, 239, 239, 0);
	padding: 2px;
}


QPushButton 
{
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(200, 200, 200, 255), 
stop:0.5 rgba(221, 221, 221, 255),
stop:1 rgba(200, 200, 200, 255));
	border-style: outset;
    border-width: 2px;
    border-color: rgba(234, 234, 234, 255)
 }

QPushButton#pushButton_02_buyitnow
{
	background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(234, 0, 0, 255), 
stop:0.5 rgba(255, 39, 39, 255),
stop:1 rgba(234, 0, 0, 255));
	color: rgb(249, 249, 249);
}

 QPushButton:pressed {
	background-color: rgb(221, 221, 221);
 }

 QPushButton:hover {
	 background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
stop:0 rgba(229, 229, 229, 255), 
stop:0.5 rgba(221, 221, 221, 255),
stop:1 rgba(229, 229, 229, 221));
    border-color: rgba(221, 128, 32, 255)
 }


QLineEdit
{
	background-color:  rgba(230, 230, 230, 255);
	border-style: outset;
    border-width: 2px;
    border-color: rgba(234, 234, 234, 255);
 }
</string>
  </property>
  <widget class="QLabel" name="label_thumbnail">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>160</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;Thumbnail&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>165</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;Name&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_bidprice">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>185</y>
     <width>65</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;Bid&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_buyoutprice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>185</y>
     <width>65</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;Buyout&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_01_bid">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>210</y>
     <width>65</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Bid</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_02_buyitnow">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>65</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Buy</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
